erigon-lib/downloader/downloader.go

```diff
@@ -183,6 +183,11 @@ func (r *requestHandler) RoundTrip(req *http.Request) (resp *http.Response, err
 		case http.StatusInternalServerError, http.StatusBadGateway:
 			r.downloader.stats.WebseedServerFails.Add(1)
 
+			if resp.Body != nil {
```

Comment:
maybe also add: 
```
defer func() {
   if err == nil && resp != nil && resp.Body != nil {
        resp.Body.Close()
   }
}
```

Reply:
This is an intermediate function I'm not sure it should be catching panics - I think they should propagate up to the caller ?

Reply:
If need propagate - can propagate by `panic(rec)`.  i'm only about closing body.